### ISSUE TITLE
Horizon indicator

### ISSUE THREAD

**Issue by GingerSteve** [GingerSteve/Stargazing-App#12]
Would this help, or be distracting?
Should it be optional?
How to implement - skybox, or game object of some kind?

**Comment by atlantichalibut** [comment]
I think the the Horizon indicator would be helpful because I find that in between both North and South indicators it can get somewhat confusing as to where I am. If it were to be a light and faded grey I think it would blend nicely in with the background.

I think it should be optional and let the user decide whether they find it distracting or not.

I'm thinking using a game object as it may allow for some greater functionality later on

**Comment by GingerSteve** [comment]
@atlantichalibut Go for it! I agree that it should be some type of GameObject instead of a skybox.

**Comment by atlantichalibut** [comment]
Alright so I've got a pretty decent horizon setup using a LineRenderer. There are a few tweaks I need to figure out to change transparency and the like but right now I realized I needed some colour. I was thinking the red because it's noticeable but the white seems to just blend in. Any thoughts?

![red horizon](https://user-images.githubusercontent.com/33707802/39555353-6358c602-4e3e-11e8-9a5a-7c016476283f.PNG)
![white horizon](https://user-images.githubusercontent.com/33707802/39555354-63876516-4e3e-11e8-8e4c-535a944f23a3.PNG)
